galaxy_ng/tests/integration/utils/iqe_utils.py

```diff
@@ -660,9 +664,11 @@ def has_old_credentials():
 @lru_cache()
 def get_hub_version(ansible_config):
     if aap_gateway():
-        username = ansible_config("admin").PROFILES.get("admin").get("username")
-        password = ansible_config("admin").PROFILES.get("admin").get("password")
```

Comment:
@chr-stian is there a reason it was done this way?

Reply:
I simply messed up